crypto/zk/artifacts/artifacts.go

```diff
@@ -227,6 +229,16 @@ func downloadFile(ctx context.Context, fromUrl, toPath string) error {
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("error on download file %s: %d", fromUrl, resp.StatusCode)
```

Comment:
```suggestion
		return fmt.Errorf("error on download file %s: http status is %d", fromUrl, resp.StatusCode)
```
I think indicating what the number is will make it more clear.